From ba1960120ccd284cf20d82f5001beb25c35427b2 Mon Sep 17 00:00:00 2001
From: Ismo Puustinen <ismo.puustinen@intel.com>
Date: Tue, 22 Sep 2015 16:06:10 +0300
Subject: [PATCH 24/26] spi-pxa2xx: fixed ACPI-based enumeration of SPI
 devices.

Slave devices were not enumerated by ACPI data because the ACPI handle
for the spi-pxa2xx controller was NULL if it was itself enumerated by
PCI.

Upstream-status: Inappropriate, real fix forthcoming
---
 drivers/spi/spi-pxa2xx-pci.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/spi/spi-pxa2xx-pci.c b/drivers/spi/spi-pxa2xx-pci.c
index fa7399e..c5a7111 100644
--- a/drivers/spi/spi-pxa2xx-pci.c
+++ b/drivers/spi/spi-pxa2xx-pci.c
@@ -170,6 +170,7 @@ static int pxa2xx_spi_pci_probe(struct pci_dev *dev,
 	memset(&pi, 0, sizeof(pi));
 	pi.parent = &dev->dev;
 	pi.name = "pxa2xx-spi";
+	pi.fwnode = dev->dev.fwnode;
 	pi.id = ssp->port_id;
 	pi.data = &spi_pdata;
 	pi.size_data = sizeof(spi_pdata);
-- 
2.4.3

